locations/spiders/wilko_gb.py

```diff
@@ -8,10 +8,17 @@ class WilkoGBSpider(SitemapSpider, StructuredDataSpider):
     item_attributes = {"brand": "Wilko", "brand_wikidata": "Q8002536"}
     allowed_domains = ["stores.wilko.com"]
     sitemap_urls = ["https://stores.wilko.com/sitemap.xml"]
-    sitemap_rules = [
-        (
-            r"https:\/\/stores\.wilko\.com\/gb\/([\w-]+\/[-\w\/]+)$",
-            "parse_sd",
-        ),
-    ]
+    sitemap_rules = [(r"https:\/\/stores\.wilko\.com\/gb\/([\w-]+\/[-\w\/]+)$", "parse_sd")]
     wanted_types = ["DepartmentStore"]
+
+    def post_process_item(self, item, response, ld_data, **kwargs):
+        if closed := response.xpath(
+            '//span[@class="AlertBanner-text"][contains(text(), "The store will permanently close on ")]/text()'
+        ).get():
+            closed_date = closed.replace("The store will permanently close on ", "")
+            item["extras"]["confidence:feature"] = 0.0
```

Comment:
Is `confidence:feature` an OSM tag or something we've decided upon for ATP? I can only see related `operational_status:reliability` tag for OSM per https://wiki.openstreetmap.org/wiki/Humanitarian_OSM_Tags/Humanitarian_Data_Model#Operational_status.

What does this tag intend to convey, particularly for value `0.0`? It sounds like a statement that we're 0% confident the feature exists? Which isn't _quite_ true if we're still waiting for `end_date` to occur because we're somewhat confident the feature still exists until `end_date`. Then after `end_date` we're now closer to 100% confident the feature no longer exists?

Reply:
These end dates are in the past (and un formatted as I wasn't using them), I hadn't considered an end_date in the future. I agree if we get that data, we should output it. I don't think I've seen a site give us that data yet.

Reply:
> These end dates are in the past (and un formatted as I wasn't using them), I hadn't considered an end_date in the future. I agree if we get that data, we should output it. I don't think I've seen a site give us that data yet.

I've only seen future closure dates be mentioned a few times, generally in the context of temporary closures for renovations/maintenance or an upcoming relocation to a new address. I think in all cases though, it wouldn't be worth parsing because it would have been information generally provided in freeform comments fields, hence hard to reliably extract.